examples/pt2/torch_compile_openvino/README.md

```diff
@@ -0,0 +1,146 @@
+
+# TorchServe inference with torch.compile with OpenVINO backend of vgg16 model
```

Comment:
It will be good if  this is changed to ResNet 50. We run nightly benchmarks for torch.compile with ResNet50. So, this will be an easy addition.

Reply:
Sure will change this